Assets/Game/Scripts/Player.cs

```diff
@@ -144,7 +119,7 @@ void Start()
 
         
         if (varJumpTimer) {
-            if (jumpKeyDown) {
+            if (_jumping) {
```

Comment:
nit: this should be called _jumpKeyDown instead. The name _jumping can be misleading since it suggests the character is jumping right now which is not necessarily true if the jump button is down.